<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagePeersDialog</class>
 <widget class="QDialog" name="ManagePeersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理IP地址</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblInformation">
     <property name="text">
      <string>通常不需要添加手机的IP。只有当手机和电脑的IP位于不同的网段时才需要把手机的IP添加到这里来。</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="txtIP">
       <property name="placeholderText">
        <string>192.168.1.105</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>添加(&amp;A)</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>删除被选的IP(&amp;R)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="lstIPs"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManagePeersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManagePeersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
